x/dualstaking/keeper/migrations.go

```diff
@@ -63,6 +62,12 @@ func (m Migrator) ConvertProviderStakeToSelfDelegation(ctx sdk.Context) error {
 		}
 	}
 
+	moduleBalance := m.keeper.bankKeeper.GetBalance(ctx, m.keeper.accountKeeper.GetModuleAddress(types.ModuleName), epochstoragetypes.TokenDenom)
+	err := m.keeper.bankKeeper.BurnCoins(ctx, types.ModuleName, sdk.NewCoins(moduleBalance))
```

Comment:
we are only burning dual staking, we will need to add pairing module burn in a migration (doesn't have to be in this one)

Reply:
why?
mpdule pairing after this migration shouldnt hold any coins